coverage/tomlconfig.py

```diff
@@ -40,7 +40,7 @@ def read(self, filenames):
         filename = os.fspath(filenames)
 
         try:
-            with open(filename, encoding='utf-8') as fp:
+            with open(filename, encoding='utf-8', newline='') as fp:
```

Comment:
@hukkin do you have a test file that breaks without `newline=''`?

Reply:
Any file that has a CR character not immediately followed by a LF character. The most simple example is a one byte file with just the CR character (0x0D).

That is invalid TOML and should raise an error, but doesn't if `newline=''` is not set.

Reply:
Thanks, I'll push a test case